tests/test_timer_integration.py

```diff
@@ -0,0 +1,31 @@
+"""Integration tests for the timer."""
```

Comment:
Do we need to add the license/copyright text here or it isn't needed?

Reply:
Not sure if it is legally needed but we are doing it, will add.

Reply:
Done!